ambry-router/src/main/java/com/github/ambry/router/NonBlockingRouterMetrics.java

```diff
@@ -184,7 +184,9 @@ public class NonBlockingRouterMetrics {
   public final Meter operationErrorRate;
   public final Meter encryptedOperationErrorRate;
   public final Counter slippedPutAttemptCount;
+  public final Counter metadataSlippedPutAttemptCount;
```

Comment:
Can we add comment for 
    slippedPutAttempCount,
    slippedPutSuccessCount,
    metadataSlippedPutSuccessCount
    metadataPutAttemptCount.

Looks like your intention is that slippedPutXXX includes both data chunk and meta data while metadataXXX is as name indicated.

Reply:
You are right about the intention.
Added the comments.